src/compile.rs

```diff
@@ -214,6 +218,22 @@ fn compile_target(
             ];
             rustc_args.extend(mac_args);
         }
+    } else if target.is_emscripten() {
+        shared_args.push("-Zbuild-std");
```

Comment:
Hopefully https://github.com/rust-lang/rust/pull/98149 will be merged soon and you won't have to use this.

Reply:
When that happens we can also remove `-Crelocation_model=pic`.